test/types/index.ts

```diff
@@ -324,19 +324,23 @@ server.setSchemaCompiler(function (schema: object) {
 
 server.addSchema({})
 
-server.addContentTypeParser('foo/bar', {}, (req, done) => {
+server.addContentTypeParser('*', (req, done) => {
   done!(null, {})
 })
 
-server.addContentTypeParser('foo/bar', { parseAs: 'string' }, (req, done) => {
```

Comment:
This too

Reply:
This conflicts with your docs. It says:

```
As you can see, now the function signature is (req, body, done) instead of (req, done)
```

Additionally, if this is allowed, it seems like the body is getting parsed as a string and then ignored. What's the point?